config/internal/configsource/testdata/err_begin_session.yaml

```diff
@@ -0,0 +1,4 @@
+simple:
```

Comment:
Could you change a couple of examples with the expected usage of real config sources, like Kubernetes configmaps? While I understand the code, it's a bit hard for me to translate it into real-world usage.

Reply:
I have a conversation today and the format should change a bit, when that is done I will add larger examples (besides the test files).

Reply:
Resolving since this part was removed from the PR.